projects/commudle-admin/src/app/feature-modules/public-community/components/home-community/home-community.component.scss

```diff
@@ -1,7 +1,8 @@
 @import '/projects/commudle-admin/src/assets/styles/variables';
 
 .home-component {
-  max-width: 1280px;
+  // max-width: 1280px;
+  max-width: 976px;
```

Comment:
When this PR is merged with the others, we have to check if reducing the width is affecting other things